domain/src/test/java/uk/gov/hmcts/ccd/definition/store/domain/service/EntityToResponseDTOMapperTest.java

```diff
@@ -129,6 +129,7 @@ void testMapEventCaseFieldEntity() {
             eventCaseFieldEntity.setRetainHiddenValue(true);
             eventCaseFieldEntity.setPublish(true);
             eventCaseFieldEntity.setPublishAs("PublishAs test");
+            eventCaseFieldEntity.setDefaultValue("");
```

Comment:
Worth setting this to some non-empty string value to prove it is mapping correctly
```suggestion
            eventCaseFieldEntity.setDefaultValue("DefaultValue test");
```

Reply:
Fixed in [5e6bcf3](https://github.com/hmcts/ccd-definition-store-api/pull/1111/commits/5e6bcf3530b16e5bff37a8f0c02855a0d686d834)